src/components/dataset/new.vue

```diff
@@ -130,8 +132,19 @@ const hideOrComplete = () => {
 
 <style lang="scss">
 @import '../../assets/scss/variables';
+@import '../../assets/scss/mixins';
 
 #dataset-new-success {
+  display: flex;
+  align-items: center;
```

Comment:
If this is `align-items: center;` here, then I'm guessing that the `.icon-check-circle` doesn't need `vertical-align: middle;`.